src/EnergyPlus/WeatherManager.hh

```diff
@@ -71,6 +71,8 @@ struct EnergyPlusData;
 
 namespace Weather {
 
+    int constexpr NumDaysInYear = 365;
```

Comment:
If EndDayOfMonthWithLeapDay(2) is changed for leap year, why isn't NumDaysInYear also changed for leap year?

Reply:
I have no idea. I'm just shuffling the deck chairs here. Should've just stuck with initializing the few variables that weren't instead of going whole hog on this header.

Reply:
Yeah, I wasn't suggesting to change anything here. But it did cross my mind that maybe we missed something with leap year. Specifically, the number of days in the year.

Reply:
> If EndDayOfMonthWithLeapDay(2) is changed for leap year, why isn't NumDaysInYear also changed for leap year?

@jasondegraw Any enlightenment you can offer here?

Reply:
> > If EndDayOfMonthWithLeapDay(2) is changed for leap year, why isn't NumDaysInYear also changed for leap year?
> 
> @jasondegraw Any enlightenment you can offer here?

@rraustad `EndDayOfMonthWithLeapDay` was added by @JasonGlazer in #10185. So, it's quite possible that `NumDaysInYear` *should* be changing with leap years, but that's going to require some further investigation. I'm going to revert the changes here that move `NumDaysInYear` to a constexpr so that we don't have to undo all that later if we decide that is should change to 366 for leap years.